src/chrome/content/rules/Wikimedia.xml

```diff
@@ -70,13 +77,37 @@ Fetch error: http://icinga.wmflabs.org/ => https://icinga.wmflabs.org/: (6, 'Cou
 	<target host="wiktionary.org" />
 	<target host="*.wiktionary.org" />
 
-	<!-- Wikimedia chapters -->
-	<target host="wikimedia.ca" />
-	<target host="www.wikimedia.ca" />
+	<test url="http://en.wikibooks.org" />
+	<test url="http://en.wikinews.org" />
+	<test url="http://en.wikipedia.org" />
+	<test url="http://en.wikiquote.org" />
+	<test url="http://en.wikisource.org" />
+	<test url="http://en.wikiversity.org" />
+	<test url="http://en.wikivoyage.org" />
+	<test url="http://en.wiktionary.org" />
+	<test url="http://de.wikibooks.org" />
+	<test url="http://de.wikinews.org" />
+	<test url="http://de.wikipedia.org" />
+	<test url="http://de.wikiquote.org" />
+	<test url="http://de.wikisource.org" />
+	<test url="http://de.wikiversity.org" />
+	<test url="http://de.wikivoyage.org" />
+	<test url="http://de.wiktionary.org" />
+	<test url="http://ru.wikibooks.org" />
+	<test url="http://ru.wikinews.org" />
+	<test url="http://ru.wikipedia.org" />
+	<test url="http://ru.wikiquote.org" />
+	<test url="http://ru.wikisource.org" />
+	<test url="http://ru.wikiversity.org" />
+	<test url="http://ru.wikivoyage.org" />
+	<test url="http://ru.wiktionary.org" />
+
+	<test url="http://www.wikidata.org" />
+	<test url="http://www.wikimedia.org" />
+	<test url="http://www.wikimediafoundation.org" />
 
 	<!-- Wikimedia Tool Labs -->
 	<target host="tools.wmflabs.org" />
-	<target host="icinga.wmflabs.org" />
```

Comment:
Hm, I think this one timed out or errored when I tried it, but now seems fine. I'll add it back in.